geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/AbstractIngest.scala

```diff
@@ -70,6 +70,24 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
 
   val ds = DataStoreFinder.getDataStore(dsParams)
 
+  // TODO modify to use dual callbacks...now we have reduce phases
+  case class MapOnlyCallback(startTime: Long,
```

Comment:
should we remove these for now, or are you planning on implementing something for this pr?